dhis-2/dhis-support/dhis-support-test/src/main/resources/tracker/deduplication/potential_duplicates.json

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
it's definitely easier to understand the potential duplicate tests if we use this json, but didn't we say we want to have one json for metadata and one for data only?

Reply:
I am not sure that every tests requirements should affect all other tests 😅 All the exporter tests we have of which there are many do not need these TEs.